integrations/cross_platform/yocto.rst

```diff
@@ -0,0 +1,213 @@
+.. _yocto_integration:
+
+
+|yocto_logo| Yocto
+__________________
+
+The `Yocto Project`_ is an open-source project that delivers a set of tools that create operating system images for embedded Linux systems.
+The Yocto Project tools are based on the `OpenEmbedded`_ project, which uses the BitBake build tool, to construct complete Linux images.
+
+Brief introduction to Yocto
+===========================
+
+Yocto supports several Linux host distributions and it also provides a way to install the correct version of these tools by either
+downloading a buildtools-tarball or building one on a supported machine. This allows virtually any Linux distribution to be able to run
+Yocto, and also makes sure that it will be possible to replicate your Yocto build system in the future. The Yocto Project build system also
+isolates itself from the host distribution's C library, which makes it possible to share build caches between different distributions and
+also helps in future-proofing the build system.
+
+Creating Conan packages with Yocto's SDK toolchain
+==================================================
+
+Yocto SDKs are completely self-contained. The binaries are linked against their own copy of `libc`, which results in no dependencies on the
+target system.
```

Comment:
Mhh, I think this is confusing. The idea is, you are linking with the `libc` of the `host` but in your `build` machine. Mention the idea that the SDK is a cross-building toolchain matching the target and generated from a specific "yocto project" (find other words), so if you specify/enable some system dependency in the yocto project and then you generate an SDK, you will have these libraries/tools available in the SDK, e.g: GTK